### ISSUE TITLE
1.10.2?

### ISSUE THREAD

**Issue by eksynn** [EnviroMine/EnviroMine-1.7#217]
any chance for a 1.10.2 final release? ^^;;

**Comment by NightEule5** [comment]
Yes, there is a slim chance I will be able to update it to 1.11. https://twitter.com/Wise_Owl51/status/834607263461445633

Not going to make any promises that I will finish though.

**Comment by RoboMedZi** [comment]
Good luck, I really loved this mod, it was one of the few mods that made things difficult, but interesting and realistic. It's great to see you bring it back.

Although, I think it would be nice if you make either the default status bars match with the enviromine bars or make the enviromine bars match with the default status bars eventually.

**Comment by NightEule5** [comment]
Thanks for the suggestion. I will try to add to that.

**Comment by eksynn** [comment]
any news on a 1.13 update? =D

also... how do i download this? i don't get github o- o

**Comment by NightEule5** [comment]
Well... I haven't touched it in I don't know how long. I've stepped away, at least for now due to my course load this semester and, to be honest, getting bored of it. I'm sorry about any inconvenience. :(

I don't know if I'll continue updating it or not, maybe someone else will...?
